app/api/recommend.py

```
@@ -133,7 +148,7 @@ def hybrid_recommend(
     # 3. 추천 결과 계산
     results = recommender.get_hybrid_scores(db, user_id, user_vec)
     recommended_brand_ids = [brand_id for brand_id, _ in results]
-    logger.info(f"Recommendation results for user {user_id}: {results}")
+    logger.debug(f"[RECO][DONE_BRAND] endpoint={endpoint} userId={user_id} resultCount={len(results)}")
```

Ruff (0.12.2)

Undefined name `endpoint`

(F821)